Modules/System Tests/Azure Blob Services API/src/ABSTestLibrary.Codeunit.al

```diff
@@ -30,6 +30,23 @@ codeunit 132921 "ABS Test Library"
         exit(Any.AlphabeticText(Any.IntegerInRange(5, 50)));
     end;
 
+    procedure GetBlobTags(): Dictionary of [Text, Text]
+    var
+        TagDictionary: Dictionary of [Text, Text];
+        i: Integer;
+    begin
+        for i := 1 to 10 do begin
+            Any.SetSeed(Random(2000));
```

Comment:
Wouldn't this be a bit too random 😄

Reply:
What do You mean by that? :) When drawing these values, I followed the method GetBlobName from the same Codeunit.